### ISSUE TITLE
Fix invalid HTML on OV page

### ISSUE THREAD

**Issue by JeffAbrahamson** [transport-nantes/tn_web#193]
The [OV page](https://beta.mobilitains.fr/observatoire/nantes/velopolitain) still has invalid html.

![image](https://user-images.githubusercontent.com/1220683/133939375-68a1ccf0-faea-4175-9884-bdd023a372ff.png)

Make a new django project as a MRE for SO and/or the package site, push the repo with a good test name, and ask on SO or as an issue on the package's site, whichever seems more appropriate.  (Look for evidence of questions with answers to make your decision.)

In asking the question, provide context and code in your question -- don't require the reader to go to the repo.  It's there only as backup.  In addition, it's not part of the question site, so it's good practice not to make the question depend on something so exterior.

**Comment by Shriukan33** [comment]
Could you check on your side if this is still happening ? 
On the link you provide to beta, it looks like the <DOCTYPE html> and <head> are gone. 
![image](https://user-images.githubusercontent.com/70256364/135093098-e7cabd7d-36b8-4091-8383-ecae1e4b6d57.png)

Possibly it has to do with the dependabot merges ?

**Comment by JeffAbrahamson** [comment]
Yes, it looks the same.  At line 81 (now, was 141) there's a second `<!DOCTYPE html>`.  That should only appear once, at the beginning, at line 1.

**Comment by Shriukan33** [comment]
Indeed, I can see it with wget, the browser console somehow might get rid of it.

**Comment by JeffAbrahamson** [comment]
Did you ask a question on SO, provide a MRE, receive an answer?  This seems really hackish and worthy of extensive comments about everything you tried that didn't work, what you attempted, etc.  So that the next person to come along and fret about the broken HTML that is being modified by simple string substitution can at least start where you left off.

> Indeed, I can see it with wget, the browser console somehow might get rid of it.

The thing that is served is an HTML document, on which the browser console offers a particular view whose main objective is to help you see the correspondence between visual artefacts and the DOM.  The browser console is a useful tool, but if it's your only tool, you'll spend your life hammering nails with screwdrivers.

**Comment by JeffAbrahamson** [comment]
I made [this](https://github.com/transport-nantes/map-mre) test project, which is a bit better.  I'm having trouble applying what I learned back to tn_web.  I'll have to come back to it.
